appstudio-utils/Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN dnf -y --setopt=tsflags=nodocs install \
     git \
     skopeo \
     make \
+    go-toolset \
```

Comment:
Can we install just `golang`? It looks like `go-toolset` only adds the `delve` debugger and nothing else